pkg/security/secl/model/unmarshallers_linux.go

```diff
@@ -1253,6 +1253,33 @@ func (e *VethPairEvent) UnmarshalBinary(data []byte) (int, error) {
 	return cursor, nil
 }
 
+// UnmarshalBinary unmarshalls a binary representation of itself
+func (e *AcceptEvent) UnmarshalBinary(data []byte) (int, error) {
+	read, err := UnmarshalBinary(data, &e.SyscallEvent)
+	if err != nil {
+		return 0, err
+	}
+
+	if len(data)-read < 20 {
+		return 0, ErrNotEnoughData
+	}
+
+	var ipRaw [16]byte
+	SliceToArray(data[read:read+16], ipRaw[:])
+	e.AddrFamily = binary.NativeEndian.Uint16(data[read+16 : read+18])
+	e.Addr.Port = binary.BigEndian.Uint16(data[read+18 : read+20])
+
+	// readjust IP size depending on the protocol
+	switch e.AddrFamily {
+	case 0x2: // unix.AF_INET
```

Comment:
Any reason to not use directly the golang const ? ex : `case unix.AF_INET`

Reply:
Not really. Changed to the golang constant here and in other events that were using the same hardcoded values.